web/config.php

```diff
@@ -1,10 +1,23 @@
 <?php
 
+/*
+ * ************** CAUTION **************
+ *
+ * DO NOT EDIT THIS FILE as it will be overridden by Composer as part of
+ * the installation/update process. The original file resides in the
+ * SensioDistributionBundle.
+ *
+ * ************** CAUTION **************
```

Comment:
Changes in this file must be reverted, as the demo contains a customized version (see #314 for an issue about not requiring manual reverts each time though)